src/runtime/entries/cloudflare.ts

```diff
@@ -60,3 +67,28 @@ const baseURLModifier = (request: Request) => {
 function normalizeOutgoingHeaders (headers: Record<string, string | string[] | undefined>) {
   return Object.entries(headers).map(([k, v]) => [k, Array.isArray(v) ? v.join(',') : v])
 }
+
+function toNodeListener (app: App, context: H3EventContext): NodeListener {
+  const toNodeHandle: NodeListener = async function (req, res) {
+    const event = createEvent(req, res)
+    event.context.cloudflare = context
```

Comment:
Should this be:

```suggestion
    event.context.cloudflare = context.cf
```

Reply:
I think the answer to this depends on where you'll want the [fetch event context](https://developers.cloudflare.com/workers/runtime-apis/fetch-event/#parameters) to live when module syntax support lands in #681.